MVCDemos/ViewModels/DropDownButtonViewModel.cs

```diff
@@ -14,5 +14,10 @@ public class DropDownButtonViewModel {
         public IEnumerable<SimpleData> FontSizes { get; set; }
         public IEnumerable<SimpleData> LineHeights { get; set; }
         public IEnumerable<ExtendedListItemData> ProfileSettings { get; set; }
+        public IEnumerable<ListType> ListTypes { get; set; }
```

Comment:
We cannot extend this model because it's used for DropDownButton demo. Seems like we should create a new one.

Reply:
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/22b5a0be01fc7aac3b752346e4897831a0ccfafe
https://github.com/DevExpress/devextreme-demos/pull/2004/commits/aecf62acb745d5b6f5750f7d85a550e527cd7692